packages/wasmer/wasmer.1.0.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+
+authors: ["rajdakin"]
+maintainer: "engineering@wasmer.io"
+license: "MIT"
+homepage: "https://github.com/wasmerio/wasmer-ocaml"
+doc: "https://github.com/wasmerio/wasmer-ocaml"
+bug-reports: "https://github.com/wasmerio/wasmer-ocaml/issues"
+dev-repo: "git+https://github.com/wasmerio/wasmer-ocaml.git"
+tags: ["org:wasmer"]
+url {
+	src: "https://github.com/wasmerio/wasmer-ocaml/archive/refs/tags/1.0.0.zip"
+	checksum: [] # Placeholder, TODO
```

Comment:
Every release archives have to have an accompagnied checksum to be merged.

Reply:
Fixed.